examples/llm-api/ray/current_node_gpu_type.py

```
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import sys
+
+import torch
+
+
+def get_gpu_info():
+    """Get information about all available GPUs and their SM versions"""
```

Ruff (0.12.2)

First line should end with a period, question mark, or exclamation point

Add closing punctuation

(D415)